cli/command/node/update.go

```diff
@@ -28,9 +28,9 @@ func newUpdateCommand(dockerCli command.Cli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVar(&options.role, flagRole, "", `Role of the node ("worker"|"manager")`)
-	flags.StringVar(&options.availability, flagAvailability, "", `Availability of the node ("active"|"pause"|"drain")`)
-	flags.Var(&options.annotations.labels, flagLabelAdd, "Add or update a node label (key=value)")
+	flags.StringVar(&options.role, flagRole, "", `Role of the node ("worker|manager")`)
+	flags.StringVar(&options.availability, flagAvailability, "", `Availability of the node ("active|pause|drain")`)
```

Comment:
I'm slightly on the fence on these (but admitted: we weren't very consistent);  slightly wondering what's clearer on intent;

- `"foo"|"bar"` either `"foo"` (string) or `"bar"` (string)
- `"foo|bar"` a string of either foo or bar
- some other notation? (`"foo", "bar", or "baz"`) ?
- extended notation for all of these (multiple lines; short explanation for each option)


/cc @dvdksn @vvoland @usha-mandya

Reply:
in the [build docs](https://docs.docker.com/build/cache/backends/local/) we've used comma-separated notation, and the options wrapped in code spans. My subjective opinion is that comma is more readable than pipes, but this is probably debatable.

Reply:
IMO single quotes containing all options are confusing (`"foo|bar"`). 
Personally I find commas clearer than pipes when only one value can be given at once, so 3rd option looks good for me:

```go
flags.StringVar(&options.role, flagRole, "", `Role of the node ("worker" or "manager")`)
flags.StringVar(&options.availability, flagAvailability, "", `Availability of the node ("active", "pause" or "drain")`)
```